static/js/l10n/place.json

```diff
@@ -0,0 +1,118 @@
+  },
+  "chart_metadata:dotted_line_explanation": {
+    "defaultMessage": "(dotted line denotes missing data)",
+    "description": "Text to explain that dotted lines mean there are missing data."
+  },
+  "chart_metadata:explore_more": {
+    "defaultMessage": "Explore More ›",
+    "description": "Hyperlink text to explore the data in a different page."
+  },
+  "chart_metadata:export": {
+    "defaultMessage": "Export",
+    "description": "Hyperlink text to export the data shown in charts."
+  },
+  "chart_metadata:feedback": {
+    "defaultMessage": "Feedback",
+    "description": "Text label for hyperlink to give Data Commons feedback on something on our website."
+  },
+  "child_places_menu:places_in_place": {
+    "defaultMessage": "Places in {place}",
+    "description": "Used for the child places navigation sidebar. Shows a list of place contained in the current place."
+  },
+  "download_chart_link": {
+    "defaultMessage": "Download Chart Image",
+    "description": "Text for the hyperlink text that will download the chart image."
+  },
+  "download_csv_link": {
+    "defaultMessage": "Download Data as CSV",
+    "description": "Text for the hyperlink text that will download the data as a CSV."
+  },
+  "export_chart_link": {
+    "defaultMessage": "Export this chart",
+    "description": "Text for the hyperlink text that will let users export data and export charts."
+  },
+  "header:overview": {
+    "defaultMessage": "Overview",
+    "description": "Text for header or subheader of Overview charts on place pages."
+  },
+  "link:return_to_overview": {
+    "defaultMessage": "Back to overview ›",
+    "description": "Text for the link present in subtopic place pages directing the user back to the Overview page."
+  },
+  "more_charts": {
+    "defaultMessage": "More charts ›",
+    "description": "Link to explore more charts about a particular domain, such as Education or Health."
+  },
+  "place_breadcrumb": {
+    "defaultMessage": "A {placeType} in",
+    "description": "Gives context for where this place is located. E.g. Tokyo is a city in Japan, Asia."
+  },
+  "place_page_table:data_from": {
+    "defaultMessage": "Data from <a1>census.gov</a1>, <a2>fbi.gov</a2> and <a3>bls.gov</a3>",
+    "description": "The source citation for the ranking table in the Place Page."
+  },
+  "place_page_table:ranking_in": {
+    "defaultMessage": "Rankings (in)",
+    "description": "The name of the rankings column of the ranking table in the Place Page."
+  },
+  "place_page_table:ranking_value": {
+    "defaultMessage": "{rank} of {total}",
+    "description": "The main content of the ranking table, telling users that the current place's rank among other places of the same type in the same parent place. E.g. USA is 1 of 195 for country GDP."
+  },
+  "plural_places": {
+    "defaultMessage": "Places",
+    "description": "General collection of places. Used for comparison charts of Places in California."
```

Comment:
We might need a few of these, and they could be translated in different ways. Can we give it a more descriptive id that includes context?